drivers/SmartThings/zigbee-thermostat/src/vimar/init.lua

```diff
@@ -0,0 +1,79 @@
+-- Copyright 2022 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local clusters        = require "st.zigbee.zcl.clusters"
+local Thermostat      = clusters.Thermostat
+local ThermostatControlSequence = Thermostat.attributes.ControlSequenceOfOperation
+
+local capabilities              = require "st.capabilities"
+local ThermostatMode            = capabilities.thermostatMode
+
+local VIMAR_SUPPORTED_THERMOSTAT_MODES = {
+  [ThermostatControlSequence.COOLING_ONLY]                    = { ThermostatMode.thermostatMode.off.NAME,
+                                                                  ThermostatMode.thermostatMode.cool.NAME},
+  [ThermostatControlSequence.HEATING_ONLY]                    = { ThermostatMode.thermostatMode.off.NAME,
+                                                                  ThermostatMode.thermostatMode.heat.NAME}
+}
+
+local VIMAR_THERMOSTAT_FINGERPRINTS = {
+  { mfr = "Vimar", model = "WheelThermostat_v1.0" }
+}
+
+
+local vimar_thermostat_can_handle = function(opts, driver, device)
+  for _, fingerprint in ipairs(VIMAR_THERMOSTAT_FINGERPRINTS) do
+    if device:get_manufacturer() == fingerprint.mfr and device:get_model() == fingerprint.model then
+        return true
+    end
+  end
+  return false
```

Comment:
just a nit, but since this sub-driver only has one FP, you don't need to loop

you can just do:
```suggestion
  return device:get_manufacturer() == VIDAR_THERMOSTAT_FINGERPRINTS.mfr and device:get_model() == VIDAR_THERMOSTAT_FINGERPRINTS.model
```
with appropriate restructuring of `VIMAR_THERMOSTAT_FINGERPRINTS`